src/Umbraco.Tests/PublishedContent/PublishedContentTests.cs

```diff
@@ -7,20 +7,28 @@
 using Umbraco.Core.Models;
 using Umbraco.Core.Models.PublishedContent;
 using Umbraco.Core.PropertyEditors;
+using Umbraco.Tests.PublishedContent.StronglyTypedModels;
 using Umbraco.Tests.TestHelpers;
 using Umbraco.Web;
 using Umbraco.Web.Models;
 
 namespace Umbraco.Tests.PublishedContent
 {
-	/// <summary>
-	/// Tests the methods on IPublishedContent using the DefaultPublishedContentStore
-	/// </summary>
-	[TestFixture]
+    /// <summary>
+    /// Tests the methods on IPublishedContent using the DefaultPublishedContentStore
+    /// </summary>
+    [TestFixture]
     public class PublishedContentTests : PublishedContentTestBase
-	{
+    {
         private PluginManager _pluginManager;
 
+        private class StrongModel : PublishedContentModel

```

Comment:
Did you check this in by accident?

Reply:
Possibly - I think VS might have changed the formatting, but it is part of the Unit tests.

Reply:
Sure because it's in the file, but it's completely unused! ;-)